src/sonic-host-services/scripts/hostcfgd

```diff
@@ -185,6 +247,104 @@ class AaaCfg(object):
         if modify_conf:
             self.modify_conf_file()
 
+    def handle_radius_source_intf_ip_chg(self, key, keys):
```

Comment:
keys is not used.

Reply:
Yes, the fields of the key are not used directly, since we have to get an IP address (v4 or v6) to use. It can be removed.

Reply:
Are you removing "keys" ?